docs/sources/setup/install/helm/install-microservices/_index.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Install the microservice Helm chart 
+menuTitle: Install microservice Loki
+description: Installing Loki in microservice (distributed) mode using the Helm chart.
+aliases:
+  - ../../../installation/helm/microservice/
+  - ../../../installation/helm/install-microservice/
+weight: 300
+keywords: 
+---
+
+# Install the microservice Helm chart
+<!-- vale Grafana.Quotes = NO -->
+<!-- vale Grafana.Quotes = YES -->
+
+This Helm Chart deploys Grafana Loki on Kubernetes.
+
+This chart configures Loki to run Loki in [microservice / distributed mode]({{< relref "../../../../get-started/deployment-modes#microservices-mode" >}}), highly available architecture designed to work with AWS S3 object storage. 
+
+The default Helm chart deploys the following components:
+- **Ingester component** (3 replicas): Handles ingestion of data.
+- **Querier component** (3 replicas, maxUnavailable: 2): Processes queries. Up to 2 replicas can be unavailable during updates.
+- **QueryFrontend component** (2 replicas, maxUnavailable: 1): Manages frontend queries. Up to 1 replica can be unavailable during updates.
+- **QueryScheduler component** (2 replicas): Schedules queries.
+- **Distributor component** (3 replicas, maxUnavailable: 2): Distributes incoming requests. Up to 2 replicas can be unavailable during updates.
+- **Compactor component** (1 replica): Compacts and processes stored data.
+- **IndexGateway component** (2 replicas, maxUnavailable: 1): Handles indexing. Up to 1 replica can be unavailable during updates.
+
+
+<!--TODO - Update when meta-monitoring chart releases-->
```

Comment:
FYI - It has been released, just not publicized yet.

https://github.com/grafana/meta-monitoring-chart/releases

Reply:
This is really cool!! I noticed we had a monitoring and alerting section in deployments. I have linked to that for now. I can create a new PR with updated instructions for that section of the docs with the repo instructions